src/locales/en/pages.yml

```diff
@@ -57,6 +57,66 @@ rti-p60-earnings-above-pt:
+  content:
+    - Tell us the amount in the box labelled ‘postgraduate loan deductions.’ You’ll find this in the ‘other details’ section of your P60.
+    - Enter the amount in pounds without the pence, exactly as it appears on your P60.
+  details:
+    summaryText: If you have more than one P60 in a tax year
+    html: >-
+      <p>You only need to enter the amount from one P60.</p>
+rti-p60-statutory-shared-parental-pay:
+  title: Enter the ‘statutory shared parental pay’ amount from your P60
+  inset: Use your P60 from the tax year {{ yearRangeStart }} to {{ yearRangeEnd }} to answer this question.
+  content:
+    - Tell us the amount in the box labelled ‘statutory shared parental pay.’ You’ll find this in the ‘statutory payments’ section of your P60.
+    - Enter the amount in pounds and pence, exactly as it appears on your P60.
+  details:
+    summaryText: If you have more than one P60 in a tax year
+    html: >-
+      <p>You only need to enter the amount from one P60.</p>
+rti-p60-statutory-adoption-pay:
+  title: Enter the ‘statutory adoption pay’ amount from your P60
+  inset: Use your P60 from the tax year {{ yearRangeStart }} to {{ yearRangeEnd }} to answer this question.
+  content:
+    - Tell us the amount in the box labelled ‘statutory adoption pay.’ You’ll find this in the ‘statutory payments’ section of your P60.
+    - Enter the amount in pounds and pence, exactly as it appears on your P60.
+  details:
+    summaryText: If you have more than one P60 in a tax year
+    html: >-
+      <p>You only need to enter the amount from one P60.</p>
+rti-p60-statutory-maternity-pay:
+  title: Enter the ‘statutory maternity pay’ amount from your P60
+  inset: Use your P60 from the tax year {{ yearRangeStart }} to {{ yearRangeEnd }} to answer this question.
+  content:
+    - Tell us the amount in the box labelled ‘statutory maternity pay.’ You’ll find this in the ‘statutory payments’ section of your P60.
+    - Enter the amount in pounds and pence, exactly as it appears on your P60.
+  details:
+    summaryText: If you have more than one P60 in a tax year
+    html: >-
+      <p>You only need to enter the amount from one P60.</p>
+rti-p60-student-loan-deductions:
+  title: Enter the ‘student loan deductions’ amount from your P60
+  inset: Use your P60 from the tax year {{ yearRangeStart }} to {{ yearRangeEnd }} to answer this question.
+  content:
+    - Tell us the amount in the box labelled ‘student loan deductions.’ You’ll find this in the ‘other details’ section of your P60.
+    - Enter the amount in pounds without the pence, exactly as it appears on your P60.
+  details:
+    summaryText: If you have more than one P60 in a tax year
+    html: >-
+      <p>You only need to enter the amount from one P60.</p>
+rti-p60-employee-ni-contributions:
+  title: Enter the ‘employee’s contributions’ amount from your P60
+  inset: Use your P60 from the tax year {{ yearRangeStart }} to {{ yearRangeEnd }} to answer this question.
```

Comment:
Wonder if we could look at extracting these 'shared' bits of content so we only have it defined/need updating in one place?

Reply:
With the way the dynamic pages work this is not as easy to do as we would like - agreed we will leave it for now